src/public/modules/forms/admin/directives/view-feedback.client.directive.js

```diff
@@ -121,14 +114,13 @@ function viewFeedbackDirective(
         }
 
         $scope.exportCsv = function () {
-          let params = {
-            formId: $scope.myform._id,
-            formTitle: $scope.myform.title,
-          }
+          const formId = $scope.myform._id
+          const formTitle = $scope.myform.title
 
           $scope.csvDownloading = true
-          FormFeedback.downloadFeedback(params).finally(function () {
+          FormFeedback.downloadFeedback(formId, formTitle).finally(function () {
             $scope.csvDownloading = false
+            $scope.$apply()
```

Comment:
is the canonical approach here to use `$q` or `$scope`? I'm not sure

Reply:
used $q.when throughout